website/docs/language/settings/backends/configuration.mdx

```diff
@@ -1,5 +1,7 @@
 ---
 page_title: Backend Configuration - Configuration Language
+description: >-
+  Use the `backend` block to control where Terraform stores state. Learn about the available state backends, the backend block, initializing backends, partial backend configuration, changing backend configuration, and unconfiguring a backend.
```

Comment:
```suggestion
  Use the `backend` block to control where Terraform stores state. Learn about configuring and managing backends.
```

just an idea to shorten this one a bit.

Reply:
Jeffrey's examples were verbose so they hit SEO keywords and site search terms (which is where these descriptions are used). It's awkward, but his suggestions seem to have merit.